src/ui/flowgraph/datasource.h

```diff
@@ -2,20 +2,11 @@
 #ifndef DATASOURCE_H
 #define DATASOURCE_H
 
-#include "../flowgraph.h"
-
 namespace DigitizerUi {
 
-class DataSource final : public Block {
+class DataSource {
```

Comment:
Re-use one of: 
https://github.com/fair-acc/graph-prototype/blob/main/blocks/basic/include/gnuradio-4.0/basic/clock_source.hpp
https://github.com/fair-acc/graph-prototype/blob/main/blocks/basic/include/gnuradio-4.0/basic/SignalGenerator.hpp